Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-4.2.xml

```diff
@@ -0,0 +1,541 @@
+        <ns4:element name="route" type="System.Concept"/>
+        <ns4:element name="cumaltiveMedicationDuration" type="System.Quantity"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.LaboratoryTestOrder" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.SubstanceAdverseEvent" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="reaction" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.DiagnosticStudyOrder" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="radiationDosage" type="System.Concept"/>
+        <ns4:element name="radiationDuration" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.ImmunizationIntolerance" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="reaction" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.SubstanceOrder" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="reaction" type="System.Concept"/>
+        <ns4:element name="dose" type="System.Quantity"/>
+        <ns4:element name="frequency" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+        <ns4:element name="refills" type="System.Concept"/>
+        <ns4:element name="route" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.LaboratoryTestAdverseEvent" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="reaction" type="System.Concept"/>
+        <ns4:element name="LaboratoryTestPerformed" type="QDM.LaboratoryTestPerformed"/>
```

Comment:
Since I've seen a few things like this now, I'm starting to think this is intentional.  But here are a few notes:

- The `LaboratoryTestPerformed` property is not in QDM
- The use of PascalCase instead of camelCase seems a bit inconsistent
- Normally the test that had the adverse event would be represented by the `code` (although I admit that feels strange)
- In the other place you did this (with `ProcedureAdverseEvent` and `ProcedureIntolerance`), the property was represented as a `System.Concept`.  Here it is a reference to another type -- so there is some inconsistency here.